src/main/java/at/hannibal2/skyhanni/features/garden/farming/GardenUptimeCommand.kt

```diff
@@ -0,0 +1,61 @@
+package at.hannibal2.skyhanni.features.garden.farming
+
+import at.hannibal2.skyhanni.features.garden.GardenAPI
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.ChatUtils.chat
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.tracker.SkyHanniTracker
+import java.time.LocalDate
+import kotlin.time.Duration.Companion.seconds
+
+object GardenUptimeCommand {
+    private val config get() = GardenAPI.config.gardenUptime
+    private val storage get() = GardenAPI.storage?.uptimeTracker
+
+    fun onCommand(args: Array<String>) {
+        if (!config.showDisplay) {
+            ChatUtils.userError("shgardenuptime requires 'Show Garden Uptime' to be enabled")
+        }
+
+        val dayAmount = when {
+            args.isEmpty() -> 7
+            args[0].toIntOrNull() == null -> 7
+            else -> args[0].toInt().coerceAtMost(31)
+        }
```

Comment:
Imo a more elegant way of handling this.
```suggestion
        val dayAmount = args.getOrNull(0)?.toIntOrNull()?.coerceAtMost(31) ?: 7
```